daemon/transmission-daemon.service

```diff
@@ -8,6 +8,8 @@ Type=notify
 ExecStart=/usr/bin/transmission-daemon -f --log-error
 ExecReload=/bin/kill -s HUP $MAINPID
 NoNewPrivileges=true
+ProtectSystem=full
```

Comment:
Daemon is configured to run as "transmission" user, not root, so I would expect it not having write access to "/boot, /efi, /usr and /etc" anyway. What is this good for?..

Reply:
The idea is to sandbox the application, denying it from anything it shouldn't be able to access using basically security in depth